packages/create-starknet/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "create-starknet",
+  "version": "1.0.0",
+  "description": "Create starknet apps with one command",
+  "main": "index.js",
+  "scripts": {
+    "build": "rimraf ./dist && tsc",
+    "dev": "npx nodemon",
+    "start": "pnpm run build && node dist/index.js"
+  },
+  "bin": {
+    "create-starknet": "dist/index.js"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
Please feel free to add your name (if you're comfortable with it) to get proper attribution for your work!